packages/flutter_driver/lib/src/semantics.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'message.dart';
+
+/// Enables or disables Semantics.
```

Comment:
nit: when I see capitalized words in plain English text I immediately think it's either a proper noun or a reference to a type. I'm guessing this is not a proper noun, but if it's a type, consider backquoting it: `Semantics` vs Semantics.